src/simularium/VisGeometry/index.ts

```diff
@@ -276,6 +306,88 @@ class VisGeometry {
         fcam.addInput(this.camera, "position");
         fcam.addInput(this.controls, "target");
 
+        const grid1: ButtonGridApi = this.gui.addBlade({
+            view: "buttongrid",
+            size: [2, 1],
+            cells: (x, y) => ({
+                title: [["View", "Save"]][y][x],
```

Comment:
Small thing, but maybe "Go to" instead of "View". "View" makes me think i might be looking at the camera instead of through the camera. But I know this isn't really a true UX design so could wait

Reply:
I also considered "Activate"...